### ISSUE TITLE
Can't install on PHP 8.2

### ISSUE THREAD

**Issue by alvinkonda** [FriendsOfFlarum/upload#398]
I'm receiving the following error

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laminas/laminas-diactoros 2.14.0 requires php ^7.3 || ~8.0.0 || ~8.1.0 -> your php version (8.2.16) does not satisfy that requirement.
    - flarum/core v1.8.5 requires laminas/laminas-diactoros ^2.4.1 -> satisfiable by laminas/laminas-diactoros[2.14.0].
    - flarum/core is locked to version v1.8.5 and an update of this package was not requested.
```

Is your extension supported on the PHP 8.2 version?

**Comment by jasondeewright** [comment]
I think this is more of a laminas-diactoros issue, but they have much more recent versions. I'm able to install successfully with:
- php 8.2.18
- fof/upload 1.5.4
- flarum/core 1.8.5

My  laminas/laminas-diactoros resolves to 2.26.0 and everything is copacetic.

**Comment by alvinkonda** [comment]
Thanks for the reply, I managed to solve it by updating the forum.
